src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -1,5 +1,87 @@
+"""
+Solvent Extraction Model
+
+========================
+
+Author: Arkoprabho Dasgupta
+
+The Solvent Extraction unit model is used to perform the solvent extraction unit operation.
+It represents a series of tanks, referred to as stages, through which the aqueous and organic
+phases are passed through, and then subsequent extraction of the desired components happen.
+
+This model is based on the MSContactor model, but it only accounts for the material balance.
+Any kind of energy balance, pressure balance and reaction kinetics is not considered in this 
+model. However since this model is based on MSContactor, the energy balance and pressure balances 
+have to be specified as False in the stream configuration. At present this model only takes 
+2 inlets, one stream for each of the phases.
+
+Configuration Arguments
+-----------------------
+
+The user must specify the following configurations in a solvent extraction model to be able to 
+use it.
+
+The user must specify the aqueous feed input in the "aqueous_stream" configuration, with a 
+configuration defined with it which describes the properties of the aqueous feed.
+
+The user must specify the organic feed input in the "organic_stream" configuration, with a 
+configuration defined with it which describes the properties of the organic feed.
+
+The number of stages in the solvent extraction process has to be specified by the user through 
+the "number_of_finite_elements" configuration. It takes an integer value.
+
+The material transfer can happen from either of the phases to the other. To specify the direction 
+of the transfer, the "aqueous_to_organic" configuration is to be used by the user. This is a boolean 
+configuration. The default value is True, which means the material transfer is happening from the 
+aqueous phase to the organic  phase, like in the loading operation. For scrubbing and stripping, the 
+reverse happens, so the value of the configuration will be False.
+
+Stream configurations
+---------------------
+
+Each of the feed streams has to have a dictionary which specifies the property packages and other 
+details as mentioned below.
+
+The "property_package" configuration is the property package which describes the state conditions 
+and properties of a particular stream.
+
+The "property_package_args" configuration is any specific set of arguments that has to be passed to 
+the property block for the unit operation.
+
+The user can specify the direction of the flow of the stream through the stages through the 
+configuration "flow_direction". This is a Boolean type of configuration, with two inputs, based
+on the FlowDirection package. The default value is "FlowDirection.forward" which is the forward 
+direction, and the other is "FlowDirection.backward" which is the backward direction. 
+
+Since this model is based on MSContactor, the user has to give two additional arguments of 
```

Comment:
As mentioned above, this is incorrect. The model does support these (even if you don't intend to use them right now), and the user can freely set these as they wish (assuming their property package supports the necessary properties).

Reply:
Are you talking about lines 53-56?

Reply:
I have removed this part also.